frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -55,6 +58,8 @@ export default {
       lastEntry = entry.prev
     } while (lastEntry)
     this.ephemeral.logs = logs.reverse() // chronological order (oldest to most recent)
+
+    this.ephemeral.reportError = this.$route.query.errorMsg
```

Comment:
Since this variable is never re-assigned, I don't think it's needed? You should be able to access it via `this.$route.query.errorMsg` wherever it's needed right?